crossbar/node/controller.py

```diff
@@ -416,6 +417,21 @@ def get_worker_log(self, worker_id, limit=100, details=None):
 
         return self._workers[worker_id].getlog(limit)
 
+    @wamp.register(None)
+    def sign_challenge(self,
+                       challenge_method,
+                       challenge_extra,
+                       channel_id_raw,
+                       channel_id_type='tls-unique',
+                       details=None):
+        challenge = Challenge(challenge_method, challenge_extra)
```

Comment:
This change was made to address https://github.com/crossbario/autobahn-python/pull/1501/files#r752058376 i.e. without changing autobahn-python